doc/classes/Input.xml

```diff
@@ -60,6 +60,7 @@
 			<argument index="0" name="action" type="StringName">
 			</argument>
 			<argument index="1" name="exact_match" type="bool" default="false">
+				If [code]true[/code] the event that contributes to the raw strength only matches the action if the event provided exactly matches the action event. For an action like [code]KEY_S[/code], the event [code]KEY_CONTROL + KEY_S[/code] matches if [code]exact_match[/code] is [code]false[/code] but not if it is [code]true[/code].
```

Comment:
You can't add documentation for arguments this way, it should be part of the `<description>`.

Reply:
So should I move the text to inside the `<description>` tag and leave the `<argument>` tag empty?

Reply:
Yes, `argument` tags don't support text.